python_cmake_module/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>                                                                                        
+<package format="2">                                                                                         
+  <name>python_cmake_module</name>                                                                       
+  <version>0.0.0</version>                                                                                   
+  <description>Provide CMake module with extra functionality for Python.</description>
+
+  <maintainer email="esteve@osrfoundation.org">Esteve Fernandez</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
+
+  <export>
+    <build_type>ament_cmake</build_type>
+  </export>
+</package>
```

Comment:
This package needs to buildtool export depend on whatever rosdep key provides the necessary Python headers.